assets/js/apicalls.js

```diff
@@ -0,0 +1,84 @@
+
+
+const omdbAPIkey = "5af38ba4";
+var test = document.getElementById('root');
+
+function fetchAllGhibliMovies() {
+  apicall();
```

Comment:
Are these (apicall and buildOMDBApiCall) ever used in index.js? Seems like all they do is log the outputs to console. If not, we should remove them to save on API calls.